docs/policies/rust-versions.md

```diff
@@ -7,19 +7,23 @@ which aren't always able to be on that latest version.
 
 Given UniFFI is currently developed and maintained by Mozilla staff, it should
 be no surprise that an important consideration is mozilla-central (aka, the
-main Firefox repository). While at time of writing UniFFI is not used by
-mozilla-central, this policy is unashamedly focused on ensuring it will be
-able to.
+main Firefox repository). This policy exists to ensure UniFFI can always be
+used by mozilla-central.
 
 ## Mozilla-central Rust policies.
 
-It should also come as no surprise that the Rust policy for mozilla-central
-is somewhat flexible. There is an official [Rust Update Policy Document
-](https://firefox-source-docs.mozilla.org/writing-rust-code/update-policy.html])
-but everything in the future is documented as "estimated".
+The Rust policy for mozilla-central has an official
+[Rust Update Policy Document
+](https://firefox-source-docs.mozilla.org/writing-rust-code/update-policy.html),
+the tl;dr of which is:
 
-Ultimately though, that page defines 2 Rust versions - "Uses" and "Requires",
-and our policy revolves around these.
+* There's a "Uses" version which all of Mozilla's CI uses for releasing Firefox.
+  Discover it in the most recent updates to [this meta bug on Bugzilla](https://bugzilla.mozilla.org/show_bug.cgi?id=1504858)
+
+* There's a "Requires" version, which Mozilla is committed to supporting,
+  primarily for downstream builders such as Linux distributions.
+  [Discover it here](https://searchfox.org/mozilla-central/search?q=MINIMUM_RUST_VERSION&path=python/mozboot/mozboot/util.py). This version is tested in mozilla-central's CI,
+  but nothing is shipped by mozilla with it.
```

Comment:
Can't the version just be stated here, with an HTML comment to update it in lockstep with `.circleci/config.yml` (and an equivalent comment there)?

Reply:
The repo has the churn anyways through the CircleCI config, updating two locations isn't really harder than one if you have to do it anyways.

Reply:
The repo does, but this seems redundant - a link to the canonical truth is better than a copy of today's truth? I'm slightly surprised this PR is controversial.

Reply:
If you say you have an MSRV of whatever's in Firefox, but then a contributor uses features from that and CI fails because you haven't bumped the value in CircleCI config, that's a bad experience. If you're really against having the version tested by CI in the text here, maybe you can link to the CircleCI config in addition, and leave a note saying that if the two disagree, one should just update the CircleCI config?